client/jetpack-cloud/sections/comparison/table/useProductsToCompare.ts

```diff
@@ -19,9 +19,9 @@ export const useProductsToCompare = () => {
 				productSlug: PLAN_JETPACK_FREE,
 			},
 			{
-				id: 'BACKUP',
-				name: translate( 'VaultPress Backup' ),
-				productSlug: PRODUCT_JETPACK_BACKUP_T1_YEARLY,
+				id: 'STARTER',
+				name: translate( 'Jetpack Starter' ),
```

Comment:
Should this just read "Starter" here since the others are only "Complete" and "Security"?